.travis.yml

```diff
@@ -63,6 +75,18 @@ matrix:
           env:
               - ZIP_SUFFIX=ubuntu-trusty-clang
               - SOLC_STOREBYTECODE=On
+          before_install:
+              - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+              - sudo add-apt-repository -y ppa:mhier/libboost-latest
+              - sudo add-apt-repository -y ppa:hvr/z3
+              - sudo apt-get update -qq
+          install:
+              - sudo apt-get install -qq g++-8 gcc-8
+              - sudo apt-get install -qq libboost1.67-dev
+              - sudo apt-get install -qq libleveldb1
+              - sudo apt-get install -qq libz3-dev
+              - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-8 90
```

Comment:
Above it says `compiler: clang`, but here you install gcc...

Reply:
Note that this build will still use clang (``The CXX compiler identification is Clang 5.0.0`` in the cmake output).
The newer version of gcc is installed here as well to provide the newer libstdc++ on which boost depends.

Reply:
ok, I see